Add function cleanup one file

DanMiel opened the pull request:
Add OneButton function, remove unneeded code from CD_import.py

kbwbe commented:
No. Sorry, I cannot accept this PR, as it is destroying important parts of code of my WB. The code, which you want to delete in the file importpart.py (such as the very important command 'a2p_ImportPart') is used. It cannot be deleted without damaging the WB.
So i close this PR.

DanMiel commented:
kbwbe:
I did not delete code from a2p_importpart, I deleted code from CD_importpart which is a copy of your a2p_ file and I removed the code I did not need. I will change the file names or combine the files so this confusion will not happen again. 
Dan

kbwbe commented:
Thanks. I have changed within last commit the file "CD_A2plusupdater.py" in order to work with standard file a2p_importpart.py.
So nothing has to be duplicated. Perhaps you can have a look at my last two commmits.
